plugin/federation/federation_test.go

```diff
@@ -53,3 +53,30 @@ func TestMutateConfig(t *testing.T) {
 	err = f.MutateConfig(cfg)
 	require.NoError(t, err)
 }
+
+func TestInjectSourcesNoKey(t *testing.T) {
+ 	var cfg config.Config
```

Comment:
should this also be calling `config.LoadConfig("test_data/nokey.yml")`

Reply:
Oops!